server/src/db/schema.ts

```diff
@@ -130,41 +130,41 @@ export const trip = sqliteTable("trip", {
 	geojson: text('json_undefined'),
 	createdAt: integer('created_at', { mode: 'timestamp' }).default(sql`(strftime('%s', 'now'))`),
 	updatedAt: integer('created_at', { mode: 'timestamp' }),
-	ownerDocument: text('user_id)').references(()=> user.id),
+	ownerDocument: text('user_id)').references(()=> UserTable.id),
```

Comment:
I believe we are going to need to define our relationships separately as well as reference in here like you did above, example like this https://orm.drizzle.team/docs/rqb#declaring-relations

Reply:
Yes, this has to be done